test/Fortran/Makefile.MacOS

```diff
@@ -2,7 +2,7 @@
 #  Dss-Fortran testing makefile for MacOS
 #
 
-FFLAGS=-I ../heclib/heclib_f/src/headers -m64 -g -c -O -fallow-argument-mismatch
+FFLAGS=-I ../heclib/heclib_f/src/headers -m64 -g -c -cpp -O -fallow-argument-mismatch
```

Comment:
This appears to be saying to compile as c++, compiling C as c++ is dangerous. It's backwards compatible but not a perfect match.

If a specific set of code really does need to be compiled as c++ it should be in a separate file and use c-linkage. e.g.
```
extern "C" {
...
}
```

Reply:
No, what it's actually is to run the sources through the C preprocessor before compiling. Some of the Fortran sources have #ifdef, etc... and the gfortran on my Macbook didn't use the preprocessor by default.

Reply:
aah, terrible command line switch name.